File: 07_app_development/MyProject/libtsparser/src/main/java/com/tosmart/xiexuming/libtsparser/table/ServiceDescriptionTable.java
package com.tosmart.xiexuming.libtsparser.table;

import androidx.annotation.NonNull;

import java.util.ArrayList;

/**
 * @author xiexuming
 */
public class ServiceDescriptionTable {
    private ArrayList<Service> service;
    private int versionNumber;

    @NonNull
    @Override
    public String toString() {
        return "ServiceDescriptionTable{" +
                "versionNumber=" + versionNumber +
                " ,service=" + service +
                '}';
    }

    public int getVersionNumber() {
        return versionNumber;
    }

    public void setVersionNumber(int versionNumber) {
        this.versionNumber = versionNumber;
    }

    public ArrayList<Service> getService() {
        return service;
    }

    public void setService(ArrayList<Service> service) {
        this.service = service;
    }
}

File: newpaserapp/libtosmartparse/src/main/java/com/example/libtosmartparse/table/ServiceDescriptionTable.kt
package com.example.libtosmartparse.table

import com.example.libtosmartparse.table.Service
import java.util.ArrayList

/**
 * @author xxm
 * 2022/7/17
 **/
class ServiceDescriptionTable {
    private var service: ArrayList<Service>? = null
    private var versionNumber = 0
    override fun toString(): String {
        return "ServiceDescriptionTable{" +
                "versionNumber=" + versionNumber +
                " ,service=" + service +
                '}'
    }

    fun getVersionNumber(): Int {
        return versionNumber
    }

    fun setVersionNumber(versionNumber: Int) {
        this.versionNumber = versionNumber
    }

    fun getService(): ArrayList<Service>? {
        return service
    }

    fun setService(service: ArrayList<Service>?) {
        this.service = service
    }
}
